### DISCUSSION TITLE
seek help for edgemesh's raw code

### DISCUSSION THREAD

**Discussion by zz952332446** [kubeedge/kubeedge#4181] [Q&A]
could you give some guidance or docs to help me understand edgemesg's structure? I am learning its raw code written in golang, but there are few datas in website for me to refer to. Most explanations are too simple, and it seems that they are out of date and not official.

**Comment by vincentgoat** [comment]
cc @Poorunga

**Comment by Poorunga** [comment]
@zz952332446 
edgemesh github: https://github.com/kubeedge/edgemesh
edgemesh user doc: https://edgemesh.netlify.app/
